net-fs/samba/samba-4.5.1.ebuild

```diff
@@ -204,11 +204,16 @@ multilib_src_install() {
 		systemd_dounit "${FILESDIR}"/winbindd.service
 		systemd_dounit "${FILESDIR}"/samba.service
 	fi
-	rm ${D}/usr/bin/ldb*
-	rm ${D}/usr/lib/debug/usr/bin/ldb*
-	rm ${D}/usr/lib64/samba/ldb/*
-	rm ${D}/usr/bin/tdb*
-	rm ${D}/usr/lib/debug/usr/bin/tdb*
+	rm ${D}/usr/lib*/samba/ldb/*
+	mv ${D}/usr/bin/net /tmp
```

Comment:
Should this mv be inside `if use minimal`? Also, copping a simple name like "net" into and out of /tmp sounds dangerous. Use "${T}" instead which is local to the build and thus won't collide with anything.